tests/e2e/setup/500-create-project.ts

```diff
@@ -12,15 +12,18 @@ let packages = require('../../../lib/packages');
 
 export default function() {
   const argv = getGlobalVariable('argv');
-  let createProject = null;
+  let createProject: Promise<any> = null;
 
   // This is a dangerous flag, but is useful for testing packages only.
   if (argv.noproject) {
     return Promise.resolve();
   } else if (argv.reuse) {
     // If we're set to reuse an existing project, just chdir to it and clean it.
     createProject = Promise.resolve()
-      .then(() => process.chdir(argv.reuse))
+      .then(() => expectFileToExist(join(process.cwd(), 'test-project')))
```

Comment:
Nope, in `200-create-tmp-dir.ts` it already go to `tmp-root` if `reuse` is enabled. So, `.then(() => process.chdir(argv.reuse))` here is redundant.